composer.json

```diff
@@ -11,8 +11,8 @@
     }
   ],
   "require": {
-    "php": ">=5.6.4",
-    "laravel/lumen-framework": "5.4.*"
+    "php": ">=7.2.0",
+    "laravel/laravel": "^5.6"
```

Comment:
Do we need to move this package to laravel?  If so, then the readme and repo description should be updated accordingly.

Reply:
I updated the readme. Since the backend is also using Laravel this package also needs to be Laravel too.